htdocs/mrp/class/mo.class.php

```diff
@@ -1659,6 +1659,45 @@ public function getMoChilds()
 		}
 	}
 
+	/**
+	 * Function used to return all child MOs recursively
+	 *
+	 * @return Mo[]|int[]  array of MOs if OK, -1 if KO
+	 */
+	public function getAllMoChilds($depth = 0)
```

Comment:
parameter $depth must be added into comments with
`@param int $depth   Depth for recursing loop count`